### PR TITLE
added documentation for monoprice platform

### PR DESCRIPTION
**Pull request by etsinko**
**Description:**

This is a markdown for monoprice platform that enables control of [Monoprice 6-zone amplifier](https://www.monoprice.com/product?p_id=10761)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9662

### CODE REVIEW

File: source/_components/media_player.monoprice.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+title: "Monoprice 6-Zone Amplifier"
+description: "Instructions on how to integrate Monoprice 6-Zone Home Audio Controller into Home Assistant."
+date: 2017-10-02 11:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mp-logo.svg
+ha_category: Media Player
+ha_release: 0.25
```
Comment (fabaff) [inline source/_components/media_player.monoprice.markdown]: Please update.

File: source/_components/media_player.monoprice.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "Monoprice 6-Zone Amplifier"
+description: "Instructions on how to integrate Monoprice 6-Zone Home Audio Controller into Home Assistant."
+date: 2017-10-02 11:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mp-logo.svg
```
Comment (fabaff) [inline source/_components/media_player.monoprice.markdown]: Please change the filename to `monoprice.svg`. Easier to identify if the name is there.

File: source/_components/media_player.monoprice.markdown (hunk 1 above)
Comment (etsinko) [reply source/_components/media_player.monoprice.markdown]: Which release version should I put there? 0.55?

**Comment by etsinko** [comment]
@fabaff I noticed that Monoprice logo is pure white and it is barely visible on white background. I'm not sure what you usually do in such cases but I'd suggest changing it to solid black or maybe black outline
screenshot from 2017-10-08 13 17 31
screenshot from 2017-10-08 13 17 34

.

**Comment by arsaboo** [comment]
How about something like 
image
or 
image

**Comment by etsinko** [comment]
@arsaboo I'm not sure about red because it is not one of Monoprice's colors. But I think black will work great.
screenshot from 2017-10-08 13 33 31
screenshot from 2017-10-08 13 33 26

**Comment by frenck** [comment]
@fabaff, could you please revisit your approval? Thx 👍

**Pull request merged**